### ISSUE TITLE
Design review for new release 

### ISSUE THREAD

**Issue by creyeknife** [O3Labs/OzoneWalletIOS#184]
**Inbox and portfolio:**
Inbox icon not correct (Swap to comment icon)

Reduce the size of the X icon on the inbox to 80%

The switches on the filters are invisible on which and hard to see on black

Reduce text size of the date 2pt 

Remove the blank empty space when there is no call to action on a message. 

Dialogue pops every time you open inbox (assume this is still work in progress)

**Wallet selector**

Remove the combined heading.

Change Combined to Total

Change the tracked and untracked watch address titles to 
Watch Address
Watch Address (Hidden)

As discussed last week, now that it goes straight to combined view on portfolio, perhaps we can 

Can't tap combined option. It should just dismiss the wallet selector and return to combined view.

Not sure we need delete as an option for watch address here? I think it might be better to have a manage watch address screen in the general settings for edit and delete. They are not so common actions, and probably should be hidden most of the time.
Maybe these options could just be "View details" "Hide from combined total" Convert to wallet"?

Make the title of portfolio tappable

**Settings**
When swapping wallets from settings, the page doesn't update to new wallet

Security center not yet updated to auto select wallet (TBD)

Add manage watch address to general, with options to add delete and edit watch addresses

Help page needs a little spacing between items. Lots of room on page but everything is bunched up in the corner. Add maybe 12pt more between each item and the top and left edge. 

Update referral UI to new design. (TBD)

Auto copy to clipboard if you tap the referral link box. Instead of having to manually select the address and press copy. Just like it does when you tap the QR code on settings. 

I think the general settings should be above security center. While it's not as important, the hierarchy feels wrong currently.

Security center updates TBD.

**Dark Mode**
Switches on inbox filter are too dark
Seems pretty good overall otherwise

**Extra points:**
Remove the text and button from the featured items. I don't think it's really needed. The image has the text and we can add a CTA in the images if it's really needed. but already sort of obvious anyway.

**Comment by saltyskip** [comment]
I think all options for watch address should be in that screen, they have no other purpose except viewing in explorer

**Comment by creyeknife** [comment]
I disagree, there is difference between managing and interacting and while placing them all together may be easier, I don't think it would be more intuitive for the user.

**Comment by saltyskip** [comment]
Watch addresses have no place in the app except in that one screen, why would it be more intuitive to place settings related to them somewhere else?

**Comment by creyeknife** [comment]
Because regardless of where it is used/interacted in the app, generally management tools are grouped together in the settings. 

Same as wallets, you manage them together in the settings. 

If you asked someone to delete their watch address, their first thought won't be to open the wallet selector and goto the watch address there to delete it. It would be to head to the settings I believe.

**Comment by saltyskip** [comment]
Also think we should remove unlock functionality of wtach address entirely

**Comment by creyeknife** [comment]
Yeah that’s cool I think.
